src/toro.cr

```diff
@@ -53,8 +53,10 @@ module Toro
     getter context : HTTP::Server::Context
 
     @inbox = Hash(Symbol, String).new
+    @halt = false
 
     def initialize(@context)
+      @halt = false
```

Comment:
@soveran , with this change you'll get a 30% increment in the speed of the router (benchmarked with wrk in a linux machine)